pkg/services/navtree/navtreeimpl/admin.go

```diff
@@ -154,6 +154,16 @@ func (s *ServiceImpl) getAdminNode(c *contextmodel.ReqContext) (*navtree.NavLink
 		})
 	}
 
+	if s.license.FeatureEnabled("groupsync") {
+		accessNodeLinks = append(accessNodeLinks, &navtree.NavLink{
+			Text:     "External group sync",
+			Id:       "groupsync",
+			SubTitle: "Manage mappings of Identity Provider groups to Grafana user attributes",
```

Comment:
nit: I wonder if we could paraphrase this to make it easier to understand. For instance, could we list all the "Grafana user attributes" for which we support mapping (I think it's only roles and teams, right)?

Reply:
I'm not sure I follow - I think we could add them in the future, but since it only supports a single one right now (RBAC roles) I don't know if it is helpful to enumerate that.